### DISCUSSION TITLE
Alt instead Title on links !

### DISCUSSION THREAD

**Discussion by iamluxan** [JiHong88/SunEditor#1168] [General]
Hi,
sunEditor generates "<a href="..." **alt**="Alt text">...</a>" instead <a href="..." **title**"title text">...</a>

It is a real mistake for any seo, we need title on links and not alt (alt is used for images and not links).

https://www.w3.org/TR/html401/struct/global.html#adef-title

**Comment by JiHong88** [comment]
In the current version I will remove the alt attribute and in version3 I will add a title input field.

**Reply by JiHong88** [reply]
https://github.com/JiHong88/suneditor/issues/1190

**Reply by iamluxan** [reply]
Thank you :-)
